wait_for_caluma.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Testing pipeline helper to make sure, postgres and caluma are running before
+# starting the tests
+
+set -e
+
+# Max query attempts before consider setup failed
+MAX_TRIES=5
+
+function dbIsReady() {
+  docker-compose logs db | grep "database system is ready to accept connections"
+}
+function calumaIsReady() {
+  docker-compose logs caluma | grep "spawned uWSGI http"
+}
+
+function waitUntilServiceIsReady() {
```

Comment:
In other projects, we use wait_for_it.sh from https://github.com/vishnubob/wait-for-it. This should be a little bit more robust

Reply:
Done!